.gitignore

```diff
@@ -21,7 +21,10 @@ dist/
 .env
 server/.env
 client/.env
+package-lock.json
+package.json
```

Comment:
`package.json` and `package-lock.json` shouldn't be ignored, we need them to be able to install the packages for the project.